src/DynamoCore/Utilities/LuceneSearchUtility.cs

```diff
@@ -69,9 +69,6 @@ internal void InitializeLuceneConfig(string dirName, LuceneStorage storageType =
             }
                 
 
-            // Create an analyzer to process the text
-            Analyzer = new StandardAnalyzer(LuceneConfig.LuceneNetVersion);
```

Comment:
@RobertGlobant20 @reddyashish Is this the line that needs to be removed in 2.19 branch as well for searching in language?

Reply:
Yes, that's right.

Reply:
This is addressed in Roberto's PR for 2.19. 
https://github.com/DynamoDS/Dynamo/pull/14264/files#diff-137d5f2b3a8a3d92662c216d997b8c51e6985dcc352e7a3c9c7de8a2cdb2fb54L86